app/templates/components/list-filter.hbs

```diff
@@ -1,2 +1,4 @@
-City: {{input value=value key-up=(action 'handleFilterEntry')}}
+<label for="filterByCity" class="screen-reader">Filter By City</label>
+  {{input id="filterByCity" value=value key-up=(action 'handleFilterEntry') class="light" placeholder="Filter By City" role="combobox" aria-expanded="true" aria-autocomplete="list" aria-owns="rental-list"}}
```

Comment:
so implement a keyUp event handler in the component.  Can do.  Sorry, its late :-P